ledger-core/virtual/execute/execute_test.go

```diff
@@ -834,7 +834,7 @@ func TestSendVStateReportWithMissingState_IfConstructorWasInterruptedBeforeRunne
 			return false
 		}
 	})
-	virtualTestutils.WaitSignalsTimed(t, 10*time.Second, vStateReportRecv)
+	virtualTestUtils.WaitSignalsTimed(t, 10*time.Second, vStateReportRecv)
```

Comment:
maybe WaitSignalsTimed could go to common testutils? Does it have something virtual specific?

Reply:
maybe separate PR